crates/matrix-sdk-ui/src/timeline/pinned_events_loader.rs

```diff
@@ -104,9 +104,10 @@ impl PinnedEventsLoader {
                 .unwrap_or_else(|_| MilliSecondsSinceUnixEpoch::now())
         }
 
+        // Sort using reversed chronological ordering (newest -> oldest)
         let sorted_events = loaded_events
             .into_iter()
-            .sorted_by(|e1, e2| timestamp(e1).cmp(&timestamp(e2)))
+            .sorted_by(|e1, e2| timestamp(e1).cmp(&timestamp(e2)).reverse())
```

Comment:
Isn't it simpler to do a `.rev()` after sorting all of them the normal way? 
I initially wanted to comment that the `reverse()` should go on the next line and then I realised what it's doing.

Reply:
It doesn't really matter here. You're reversing the `Ord`, the number of sorting operations will be the same.